projects/vdk-plugins/vdk-singer/src/vdk/plugin/singer/sandbox_singer.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from functools import lru_cache
+
+import pkg_resources
+import requests
+from vdk.plugin.singer.singer_command import find_packages_starting_with
+from vdk.plugin.singer.singer_command import is_likely_singer_tap
+
+
+@lru_cache(maxsize=None)
+def console_entry_points():
+    result = dict()
+    for entry_point in pkg_resources.iter_entry_points("console_scripts"):
+        parts = str(entry_point).split("=")
+        result[parts[0].strip()] = parts[1].strip()
+    return result
+
+
+@lru_cache()
+def find_singer_taps():
+    url = r"https://api.github.com/search/repositories?q=tap\-+user:singer-io"
```

Comment:
I had a look at the json. 
What exactly is this query doing it is querying for all repos from the user singer-io? 

what does   tap\-+  mean?

Can you add a comment explaining?

Reply:
This file is my experimental python file (that's why I called it sandbox_xxx) where I run experiments. It was not meant to be put in source control